crypto/derive-secret/src/lib.rs

```diff
@@ -19,7 +19,51 @@ const RAW_BYTES: &[u8; 8] = b"rawbytes";
 #[derive(Debug, Copy, Clone, Encodable, Decodable)]
 pub struct ChildId(pub u64);
 
-/// Secret key that allows deriving child secret keys
+/// `DerivableSecret` represents a secret key that can be used to derive child
```

Comment:
Some of this might also be good module-level documentation? At least the module-level docs should link here imo since it's the main struct.